rest/data/test_results.go

```diff
@@ -1 +1,68 @@
 package data
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/evergreen-ci/cedar/model"
+	dataModel "github.com/evergreen-ci/cedar/rest/model"
+	"github.com/evergreen-ci/gimlet"
+)
+
+/////////////////////////////
+// DBConnector Implementation
+/////////////////////////////
+
+
+// FindTestResultsByTaskId queries the database to find all performance
+// results with the given taskId and that fall within interval, filtered by
```

Comment:
task id

Reply:
This doc comment looks like it was copy-pasted so will need to be updated.